.travis.yml

```diff
@@ -23,7 +23,6 @@ install:
     - pip install nibabel --use-mirrors
     - pip install nose-cov --use-mirrors
     - pip install nipype --use-mirrors
-    - pip install xmltodict --use-mirrors
```

Comment:
Is this change related to the subject of the pull request ? I have the feeling that it should be in another pull request.